scripts/download/tools.mjs

```diff
@@ -160,6 +160,19 @@ export default async function main(platform) {
 
   await download(dockerURL, dockerPath, { expectedChecksum: dockerSHA });
 
+  // Download the Docker-Buildx Plug-In
+  const dockerBuildxVersion = 'v0.7.1';
+  const dockerBuildxURLBase = `https://github.com/docker/buildx/releases/download/${ dockerBuildxVersion }`;
+  const dockerBuildxExecutable = exeName(`buildx-${ dockerBuildxVersion }.${ kubePlatform }-${ cpu }`);
+  const dockerBuildxURL = `${ dockerBuildxURLBase }/${ dockerBuildxExecutable }`;
+  const dockerBuildxPath = path.join(binDir, exeName('docker-buildx'));
+  const dockerBuildxOptions = {};
+
+  if (kubePlatform !== 'darwin') {
```

Comment:
Please mention why we exclude darwin checksums ([the answer](https://github.com/docker/buildx/pull/288#issuecomment-630288471) being that notarization modifies the binaries after the checksums are generated).

Reply:
done